### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by karmelyoei**

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -31,10 +35,21 @@
...
+=======
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
+  </body>
+>>>>>>> be5d385ac1392f2f4c36b8325002281233f2acd4
```
Comment (leggomuhgreggo) [inline client/public/index.html]: Looks like there's some merge artifacts here.

File: client/src/components/Requirments/index.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+					</Grid>
+					<Grid item wrap xs={3}>
+						<div className={classes.box}>
+							<Avatar
+								variant="square"
+								src={require('../../assets/english.png')}
+								className={classes.root}
+							/>
+						</div>
+					</Grid>
```
Comment (leggomuhgreggo) [inline client/src/components/Requirments/index.js:65]: If there's time, I could see refactoring this so it maps over an array with the image paths, and returns the Grid item

File: client/src/screens/Home/index.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React, { Fragment, Component } from 'react';
 import axios from 'axios';
-import Requierments from '../../components/Requierments';
+import Requierments from '../../components/Requirments';
```
Comment (leggomuhgreggo) [inline client/src/screens/Home/index.js:3]: Requirements*

**Review by leggomuhgreggo — CHANGES_REQUESTED** [review]
Couple small fixups but looks good!

File: client/src/components/Requirments/index.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+				</Typography>
+				<Grid
+					container
+					direction="row"
+					justify="space-around"
+					alignItems="center"
+				>
+					<Grid item wrap xs={3}>
+						<div className={classes.box}>
+							<Avatar
```
Comment (RandInaim) [inline client/src/components/Requirments/index.js:31]: there is no need to wrap the avatar inside another div

File: client/src/components/Requirments/index.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+								variant="square"
+								src={require('../../assets/Fulltime.png')}
+								className={classes.root}
+							/>
+						</div>
+					</Grid>
+					<Grid item wrap xs={3}>
+						<div className={classes.box}>
+							<Avatar
```
Comment (RandInaim) [inline client/src/components/Requirments/index.js:40]: it should be a text positioned on the image.
you better do this in the refactor weeks

File: client/src/App.js [hunk 6]
```diff
@@ -20,7 +20,13 @@ function App() {
 			<Router>
 				<Switch>
 					<Route path="/" exact component={Home} />
-					<Route path="/login" exact component={Login} />
```
Comment (RandInaim) [inline client/src/App.js]: please be careful while solving the conflicts .please return it instead of the other path

**Review by RandInaim — CHANGES_REQUESTED** [review]

**Pull request merged**